app/marketing/mails.py

```diff
@@ -387,6 +388,23 @@ def tip_email(tip, to_emails, is_new):
             translation.activate(cur_language)
 
 
+def comment_email(comment, to_emails):
+
+    subject = gettext("💬 New Comment")
+
+    for to_email in to_emails:
+        cur_language = translation.get_language()
+        try:
+            setup_lang(to_email)
```

Comment:
Can move the `setup_lang` and `cur_language` calls out of the loop for a tiny bit of performance

Reply:
cur_language and setup_lang are each relative to each `to_email` so we cant do that..

Reply:
my bad on the `setup_lang` part - for the `get_language` part my understanding is that we are storing the current language (english) - then translating the email via the `setup_lang` function which changes the current language, then going back to english with the `finally:` section. So its probably good to move out of the loop, right?

Reply:
updated!